src/integrators/sdirk_integrators.jl

```diff
@@ -72,15 +67,21 @@ end
 
   cache.ηold = η
   cache.newton_iters = iter
-  u = uprev + z
+  u = u .+ dz
```

Comment:
Since mathematically we have `z <- u - uprev` initially, then an update `z <- z + dz`, at the beginning of every iteration `u <- u + dz = u + z`, at the end of every iteration `z <- z + dz`, and hence at this point (since either the loop was completely skipped or left after a full iteration - premature exit of an iteration by failed convergence is handled before) again `u <- u + dz = uprev + z`.